introduction-to-datasets.md

```diff
@@ -0,0 +1,142 @@
+
+### Names and Namespaces 
+
+URIs also define a name and namespace for your dataset. Kite uses these values when the underlying system has the same concept (for example, Hive). The name and namespace are typically the last two values in a URI. For example, if you create a dataset using the URI `dataset:hive:animals/donkeys`, Kite stores a Hive dataset named _donkeys_ in the _animals_ namespace. If you create a dataset using the URI `dataset:hdfs:/user/cloudera/animals/donkeys`, Kite stores an HDFS dataset named _donkeys_ in the _animals_ namespace.
+
+To ensure compatibility with Hive and other underlying systems, names and namespaces in URIs must be mode of alphanumeric characters or underscore (\_) and cannot start with a number.
+
+### URI Schemes
+
+You can create datasets using the schemes _Hive_, _HDFS_, _HBase_, or as local files. There are several URI patterns available and additional options. See [Dataset and View URIs][uris].
+
+Note that a URI _scheme_, which describes the storage location type, is different than a dataset _schema_, which describes the format of records in the dataset.
+
+[list]: {{site.baseurl}}/apidocs/org/kitesdk/data/Datasets.html#list(java.net.URI)
+[uris]:{{site.baseurl}}/URIs.html
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for all datasets. Kite standardizes data definition by using Avro schemas for both Parquet and Avro, and supports the standard Avro object models _generic_ and _specific_.
+
+For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
+
+```json
+{
+  "type":"record",
+  "name":"Movie",
+  "namespace":"org.kitesdk.examples.data",
+  "fields":[
+    {"name":"id","type":"int"},
+    {"name":"title","type":"string"},
+    {"name":"release_date","type":"string"},
+    {"name":"imdb_url","type":"string"}
+  ]
+}
+```
+
+The following links provide examples for inferring schemas from data files or Java classes.
+
+| Java API                                                      | Command Line Interface |
+| --------                                                      | ---------------------- |
+| [Inferring a schema from a Java Class][api-schema-from-class] | [Inferring a schema from a Java class][cli-schema-from-class] |
+| [Using the schema of an Avro data file][api-schema-from-data] | [Inferring a schema from a CSV file][cli-schema-from-csv] |
+
+[api-schema-from-class]: {{site.baseurl}}/Inferring-a-Schema-from-a-Java-Class.html
+[api-schema-from-data]: {{site.baseurl}}/Inferring-a-Schema-from-an-Avro-Data-File.html
+[cli-schema-from-class]: {{site.baseurl}}/cli-reference.html#obj-schema
+[cli-schema-from-csv]: {{site.baseurl}}/cli-reference.html#csv-schema
+
+## Partitions
```

Comment:
The other section headings refer to things that you supply to Kite: Schemas and URIs. This doesn't fit, so I think it should be changed to "Partition Strategies".